spyder_notebook/widgets/dom.py

```diff
@@ -30,7 +27,7 @@ def evaluate(self, script):
         if WEBENGINE:
             return self.page.runJavaScript("%s" % script)
         else:
-            return self.page.evaluateJavaScript("%s" % script)
```

Comment:
Oh, I took that from the Ghost.py project and leave it like as it was. But there's no problem in using `format`.

@dalthviz, please also make @goanpeca's suggested change :-)